lms/djangoapps/ccx/utils.py

```diff
@@ -210,24 +215,32 @@ def ccx_students_enrolling_center(action, identifiers, email_students, course_ke
 
     if action == 'Enroll' or action == 'add':
         ccx_course_overview = CourseOverview.get_from_id(course_key)
+        course_locator = course_key.to_course_locator()
+        staff = CourseStaffRole(course_locator).users_with_role()
+        admins = CourseInstructorRole(course_locator).users_with_role()
+
         for identifier in identifiers:
-            if CourseEnrollment.objects.is_course_full(ccx_course_overview):
-                error = _('The course is full: the limit is {max_student_enrollments_allowed}').format(
-                    max_student_enrollments_allowed=ccx_course_overview.max_student_enrollments_allowed)
-                log.info("%s", error)
-                errors.append(error)
-                break
+            must_enroll = False
             try:
-                email = get_valid_student_email(identifier)
+                email, student = get_valid_student_with_email(identifier)
+                if student:
+                    must_enroll = student in staff or student in admins or student.email == coach.email
```

Comment:
It may be better to just say `student == coach` since they should be the same object in the database (specifically they should have the same primary key, source: https://docs.djangoproject.com/en/dev/topics/db/queries/#comparing-objects)